content/en/docs/guides/air-gapped-gitops-with-ocm-and-flux.md

```diff
@@ -0,0 +1,269 @@
+
+```bash
+ocm download resource ghcr.io/phoban01//phoban.io/podinfo -c v6.3.5 config -O -
+
+apiVersion: config.ocm.software/v1alpha1
+kind: ConfigData
+metadata:
+  name: ocm-config
+...
+localization:
+- name: image # rule name
+  file: deployment.yaml # target file for substitution
+  image: spec.template.spec.containers[0].image # path in file to insert image name
+  resource: # ocm resource from which to resolve the image location
+    name: image
+```
+
+Localization contains a list of rules that describe the substitutions the `ocm-controller` needs to perform ensure that the **Local** copy of our image is deployed. OCM provides an identifier for each resource which can always be resolved to specific storage location at which the resource can be accessed. This is the secret sauce that makes it possible to automate air-gapped deployments using OCM.
+
+We can examine the image resource to see precisely where the image can be accessed:
+
+```bash
+ocm get resources ghcr.io/phoban01//phoban.io/podinfo -c 6.3.5 image -owide
+
+NAME  VERSION IDENTITY TYPE     RELATION ACCESSTYPE  ACCESSSPEC
+image 6.3.5            ociImage external ociArtifact {"imageReference":"ghcr.io/stefanprodan/podinfo:6.3.5"}
+```
+
+### Component Transfer
+
+We can use the `ocm` CLI to transfer this public component into our "private" registry. Because we are simulating an air-gapped install, we instruct the `ocm` CLI to copy the resources along with the component metadata:
+
+```bash
+AIR_GAPPED_REGISTRY=ghcr.io/phoban01/air-gapped
+
+ocm transfer component --copy-resources ghcr.io/phoban01//phoban.io/podinfo $AIR_GAPPED_REGISTRY
+```
+
+It will take few moments to complete the transfer. Once it is complete we can view the component in the air-gapped registry:
+
+```bash
+ocm get component ghcr.io/phoban01/air-gapped//phoban.io/podinfo
+
+COMPONENT         VERSION PROVIDER
+phoban.io/podinfo 6.2.3   phoban.io
+phoban.io/podinfo 6.3.5   phoban.io
+```
+
+Let's examine the image resource on the component in our private registry:
+
+```bash
+ocm get resources $AIR_GAPPED_REGISTRY//phoban.io/podinfo -c 6.3.5 image -owide
+
+NAME  VERSION IDENTITY TYPE     RELATION ACCESSTYPE  ACCESSSPEC
+image 6.3.5            ociImage external ociArtifact {"imageReference":"ghcr.io/phoban01/air-gapped/stefanprodan/podinfo:6.3.5"}
+```
+
+We can see that the image reference now points to an image that is stored in our air-gapped registry.
+
+### GitOps & Localization
+
+Now that our component has been successfully transferred lets deploy it using GitOps.
+
+We'll assume you have completed the [Deploy Applications with OCM & GitOps](/docs/guides/deploying-applications-with-gitops) guide and shall use that repository as the starting point for our air-gapped deployment.
+
+Because our air-gapped OCM repository is private we need to provide credentials. This will enable the `ocm-controller` to retrieve components from the repository.
```

Comment:
```suggestion
Because our air-gapped OCM repository is private, we need to provide credentials. This will enable the `ocm-controller` to retrieve components from the repository.
```